.github/oas.yml

```diff
@@ -0,0 +1,29 @@
+
+name: OAS
```

Comment:
This file needs to be in `.github/workflows`, otherwise it won't run I think

Reply:
:facepalm: